cypress-custom/integration/swapMod.ts

```diff
@@ -1,4 +1,4 @@
-describe('Swap', () => {
+describe('Swap (mod)', () => {
```

Comment:
I think it's convenient to add the mod, this way we know what is "custom" what is "uni" and what is "mod"